### ISSUE TITLE
Create a Sitemap of FD.io

### ISSUE THREAD

**Issue by monikappv** [frictionlessdata/project#381]
When I am planning the improvement of my website, I want to have an overview of it's structure as well as the connections between pages and subpages, so that I can understand the current structure, what may not work about it and what i need to include in any refactor

## Acceptance criteria

* [x] Sitemap in Google Drawings or similar containing all (or all important) pages/links of fd.io

## Tasks

* [x] (short) Research tools for generating (visual) sitemaps automatically - if there is a good one we can use it rather than do this by hand
* [x] Analyze the structure of fd.io
* [x] Create a sitemap
* [x] summarize (in the description maybe) your experience trying to autogenerate.

## Experience with sitemap auto generating tools
 
* Most of them generate xml sitemaps (even when the title is html sitemap generator)...
* They have restrictions on how many pages are inclued e.g. PowerMapper - first 10 pages
* form is wrong (a.k.a. tables, lists)
* they are not accurate - e.g. three branches when they have to be seven
* they are aesthetically ugly ...see all of the below 

![sitemap4](https://user-images.githubusercontent.com/56799741/70710438-daacc400-1ce7-11ea-80c0-11a0ff21e8ba.png)

![sitemap5](https://user-images.githubusercontent.com/56799741/70710609-35462000-1ce8-11ea-98a4-64f382e47c77.png)

**Comment by dipeshkhadkaa** [comment]
![Sitemap for Frictionless data](https://user-images.githubusercontent.com/55375554/70630509-cf0fbd80-1c53-11ea-86df-3ec999193b15.jpg)

**Comment by rufuspollock** [comment]
https://www.gloomaps.com/dXcfxRgyrb

![image](https://user-images.githubusercontent.com/180658/70646670-b00b2e80-1c47-11ea-8307-76b28081ad9c.png)

https://www.gloomaps.com/tG2bM23Nz4

![image](https://user-images.githubusercontent.com/180658/70647180-a7672800-1c48-11ea-9b53-c11e2356e8e0.png)

**Comment by rufuspollock** [comment]
@monikappv can you summarize (in the description maybe) your experience trying to autogenerate.

**Comment by monikappv** [comment]
> @monikappv can you summarize (in the description maybe) your experience trying to autogenerate.

@rufuspollock  summary added in the description

**Comment by rufuspollock** [comment]
FIXED. All done.
